scrapi/base/helpers.py

```diff
@@ -165,13 +172,13 @@ def oai_process_uris(*args, **kwargs):
     identifiers = gather_identifiers(args)
     provider_uris, object_uris = seperate_provider_object_uris(identifiers)
 
+    canonical_uri = (provider_uris + object_uris)
```

Comment:
I think candidates or something like that would be a better name for this, the canonicalUri variable should only ever hold the canonicalUri